src/content/reference/react/use-client.md

```diff
@@ -311,18 +312,18 @@ As `Counter` requires both the `useState` hook and event handlers to increment o
 In contrast, a component that renders UI without interaction will not need to be a Client Component.
 
 ```js
-import {getCounterValueFromFile} from 'fs-utils';
+import { readFile } from 'node:fs/promises';
 import Counter from './Counter';
 
 export default async function CounterContainer() {
-  const initialValue = await getCounterValueFromFile();
+  const initialValue = await readFile('/path/to/counter_value');
   return <Counter initialValue={initialValue} />
 }
 ```
 
-For example, the parent component of the Client Component `CounterContainer` does not require `'use client'` as it is not interactive and does not use state. In addition, `CounterContainer` must be a Server Component as it reads from the local file system on the server. This is possible because Server Components, unlike Client Components, can be async functions.
```

Comment:
we will support async functions on the client so this isn't why

Reply:
We can have async components on client?